pydm/tests/test_data_plugins_import.py

```diff
@@ -47,3 +54,23 @@ class TestPlugin1(PyDMPlugin):
 class TestPlugin2(PyDMPlugin):
     protocol = 'tst2'
 """
+
+
+def test_entrypoint_import(monkeypatch):
+    class MyTestPlugin(PyDMPlugin):
+        protocol = "__test_suite_protocol__"
+
+    class Entrypoint:
+        name = "MyTestPlugin"
+
+        def load(self):
```

Comment:
is this an emulation of the entrypoints API? It's not part of pydm, right?

Reply:
Yes, it's a mock entrypoint that allows us to do some testing. Otherwise, we'd have to have a `setup.py` and another package in the test suite which seems like overkill